Assignments/Program.cs

```diff
@@ -1,3 +1,75 @@
-﻿using System;
+﻿// ---------------------------------------------------------------------------------------
+// Spark23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// T18 - The Armstrong Number
+// ---------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using static System.ConsoleColor;
+
+namespace Spark;
+public class T18 {
+   public static void Main () {
+      List<int> task1 = new () { 0, 1, 2, 5, 9, 10, 15, 20, 25, 27 },
+                task2 = new () { 0, 1, 7, 10, 151, 370, 371, 372, 9474, 93085, 93084, 4210816, 4210818, 24678050, 24678051 };
+      Console.WriteLine ("\x1B[4m" + "Nth Armstrong Number Computer:-" + "\x1B[0m");
+      PerformTasks (task1, true);
+      Console.WriteLine ();
+      Console.WriteLine ("\x1B[4m" + "Armstrong Number Checker:-" + "\x1B[0m");
+      PerformTasks (task2, false);
+   }
+
+   /// <summary>Print the results in the console page</summary>
+   public static void PerformTasks (List<int> input, bool isTask1) {

```

Comment:
```suggestion
   public static void PerformTasks (List<int> input, bool computeNthArmstrong) {

```
Some meaningful name should be good